simulator/risc_v/riscv_instr.cpp

```diff
@@ -78,6 +78,8 @@ template<typename I> auto execute_div = ALU::riscv_div<I, sign_t<typename I::Reg
 template<typename I> auto execute_divu = ALU::riscv_div<I, typename I::RegisterUInt>;
 template<typename I> auto execute_rem = ALU::riscv_rem<I, sign_t<typename I::RegisterUInt>>;
 template<typename I> auto execute_remu = ALU::riscv_rem<I, typename I::RegisterUInt>;
+// D
+template<typename I> auto execute_slo = ALU::slo<I, typename I::RegisterUInt>;
```

Comment:
Is there a case when 2nd argument is not a I::RegisterUInt?

Reply:
No

Reply:
Then there is no need for template. Once there will be, we'll add it.